extensions/sql-database-projects/src/dialogs/addSqlBindingQuickpick.ts

```diff
@@ -194,7 +194,18 @@ export async function launchAddSqlBindingQuickpick(uri: vscode.Uri | undefined,
 							continue;
 						}
 						try {
-							connectionString = await vscodeMssqlApi.getConnectionString(connectionUri, false, false);
+							// Prompt to include password in connection string
+							let includePassword = await vscode.window.showQuickPick([constants.yesString, constants.noString], {
```

Comment:
We don't know if they have a password saved - so we should ideally be checking that before asking this (and if they don' t ask if they want to enter it manually)